content/docs/user-guide/components/reference/gameplay/tag.md

```diff
@@ -1,151 +1,55 @@
 ---
-description: ' Use the Tag component to apply labels to entities in Open 3D Engine. '
-title: Tag
+linkTitle: Tag
+title: Tag Component
+description: Use the Tag component to apply labels to entities in Open 3D Engine (O3DE).
 ---
 
+Use the **Tag** component to apply one or more labels, or *tags*, to an entity. Use these tags to find or filter entities with particular labels.
 
+## Provider
 
-Use the **Tag** component to apply one or more labels, or tags, to an entity such as **burning** or **player**. You can use these tags to find or filter entities with particular traits. For example, you can set a weapon to inflict double damage to entities tagged as **burning**.
+[O3DE Core (LmbrCentral) Gem](/docs/user-guide/gems/reference/o3de-core)
 
-## EBuses - Request Bus Interface: TagGlobalRequestBus 
+## Tag properties 
 
-Use the following request function with the **TagGlobalRequestBus** EBus interface to communicate with other components of your game.
+![Tag component properties](/images/user-guide/components/reference/gameplay/tag-component.png)
 
-For more information about using the Event Bus (EBus) interface, see [Working with the Event Bus (EBus) system](/docs/user-guide/engine/ebus/).
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Tags** | An array of tags that are added to the entity. | EntityId | None | 
 
-### RequestTaggedEntities 
+## TagHelper
```

Comment:
```suggestion
## TagComponentBehaviorHelper
```

Reply:
Confirmed that the bus is reflected as 'TagHelper' to Lua.